tests/foreman/api/test_repository.py

```diff
@@ -1521,14 +1521,22 @@ def test_positive_sync_kickstart_check_os(
         ),
         indirect=True,
     )
-    def test_missing_content_id(self, repo):
+    def test_missing_content_id(self, repo, module_sca_manifest_org, module_target_sat):
```

Comment:
We are still using sca-disabled manifest in 6.14.z and lower versions. will you please cherrypick it to 6.15.z only and have another PR for 6.14.z and lower?

Reply:
Ok I will discuss with Phoenix team and will take appropriate action.

Reply:
new PR has been opened please take a look
https://github.com/SatelliteQE/robottelo/pull/14208